openstack/networking/v2/ports/requests.go

```diff
@@ -1,9 +1,12 @@
 package ports
 
 import (
+	"errors"
 	"fmt"
 	"net/url"
 
+	"golang.org/x/exp/slices"
```

Comment:
This is in the standard library from go 1.21 so I hope it is ok to use already.
https://pkg.go.dev/slices

Reply:
what you are importing here is the experimental package that went on being added to the stdlib as `"slices"` in Go v1.21. I am bumping the Go version in https://github.com/gophercloud/gophercloud/pull/2686; once it merges you can rebase and use `"slices"` directly.

Reply:
actually, the package you import uses generics. This patch is not backportable anyway as-is.

Reply:
Please rebase on main and use `slices` directly since https://github.com/gophercloud/gophercloud/pull/2883 has merged now. Thanks!